pkg/requests/webhook_endpoints.go

```diff
@@ -40,3 +42,39 @@ func WebhookEndpointsList(ctx context.Context, baseURL, apiVersion, apiKey strin
 
 	return data
 }
+
+// WebhookEndpointCreate creates a new webhook endpoint
+func WebhookEndpointCreate(ctx context.Context, baseURL, apiVersion, apiKey, url, description string, connect bool, profile *config.Profile) error {
+	if strings.TrimSpace(url) == "" {
+		return fmt.Errorf("url cannot be empty")
+	}
+
+	data := []string{
+		"enabled_events[]=*",
+		fmt.Sprintf("url=%s", url),
+		fmt.Sprintf("api_version=%s", apiVersion),
+	}
+	if description != "" {
+		data = append(data, fmt.Sprintf("description=%s", description))
+	}
+	if connect {
+		data = append(data, "connect=true")
+	}
```

Comment:
I wonder if we should use https://pkg.go.dev/net/url#Values.Encode to url-encode this. I see this being used in other parts of the repo.

Reply:
do you mean examples like this? https://github.com/stripe/stripe-cli/blob/master/pkg/login/poll.go#L55
in that case, `client.PerformRequest` takes params as a string. 
im using `base.MakeRequest`, which takes `RequestParameters`. it will transform the params and call `client.PerformRequest` in the end